packages/compass-sidebar/src/components/non-genuine-marker.tsx

```diff
@@ -38,10 +37,9 @@ export default function NonGenuineMarker({
   return (
     <div className={nonGenuineMarkerContainer}>
       <button
-        {...focusRingProps}
```

Comment:
This would apply a class name that would be just overriden by the class name below

Reply:
are the props and behaviour of useFocusRing, etc not relevant here?

Reply:
Oh I see you changed useFocusRing as well anyway. Nevermind, then.

Reply:
Not anymore, it just returns a `className` now, [see here](https://github.com/mongodb-js/compass/pull/3424/files#diff-57f6739adcc3ac73c61e6882a02f5198a3bd269d5f5fe6ffb7f487cfa42f1bc3R79).

And for some additional context the reason for still having a hook is that in a lot of places the way just the css part was used was incorrect leading to a non-working focus ring, so when some special behavior is needed from the focus ring, the hook does the css merging in one place so that we can control better how it happens to avoid errors

Reply:
Posted this before I saw your last comment, sorry 🙈